tests/beta/test_default_tensor.py

```diff
@@ -819,6 +819,20 @@ class TestDefaultTensorIntegration:
     """Integration tests for default.tensor. This test ensures it integrates
     properly with the PennyLane interface, in particular QNode."""
 
+    @pytest.mark.parametrize("capabilities", [{"model": "qubit",
+                                               "supports_finite_shots": False,
+                                               "supports_tensor_observables": True,
+                                               "returns_probs": False,
+                                               "returns_state": False,
+                                               "supports_analytic_computation": True,
+                                               }])
```

Comment:
Is it best practice to pass through `parametrize` for just one object?

Reply:
Lol, I didn't even notice :) Yes, that makes no sense